src/olympia/promoted/utils.py

```diff
@@ -0,0 +1,49 @@
+import stripe
+
+from django.conf import settings
+
+from olympia.amo.templatetags.jinja_helpers import absolutify
+from olympia.amo.urlresolvers import reverse
+from olympia.constants.promoted import SPONSORED, VERIFIED
+
+
+def create_or_retrieve_stripe_checkout_session(subscription, customer_email):
+    if subscription.stripe_session_id:
+        return retrieve_stripe_checkout_session(subscription)
+
+    price_id = {
```

Comment:
I wonder if it makes sense to have this dict as a constant somewhere? I think similar things exist for other pieces of data.

Reply:
it's only useful in this file so I don't know. Later, it could be moved to a function, though